torcheeg/models/transformer/__init__.py

```diff
@@ -3,4 +3,5 @@
 from .vanilla_transformer import VanillaTransformer
 from .vit import ViT
 from .conformer import Conformer
-from .atcnet import ATCNet
\ No newline at end of file
+from .atcnet import ATCNet
+from .retnet import RetNet,XPOS,SimpleRetention,MultiScaleRetention
```

Comment:
Only import the final network architecture. If the user needs to access a specific module, they should import it from the corresponding file.
```
from .retnet import RetNet
```